clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -46,16 +46,16 @@ export function loadProcessorGroupCopybookPathsConfig(
   configObject: string[],
 ): string[] {
   const config = [
-    ...loadProcessorGroupSettings(item.scopeUri, "libs", [] as string[]).map(
-      (s) => normalizePath(s),
-    ),
+    ...loadProcessorGroupSettings(item.scopeUri, "libs", [] as string[]),
     ...configObject,
   ];
   return SettingsUtils.getWorkspaceFoldersPath(true)
     .map((folder) =>
       globSync(
         config.map((ele) => ele.replace(backwardSlashRegex, "/")),
         { cwd: cleanWorkspaceFolder(folder) },
+      ).map(
+        (s) => normalizePath(s),
       ),
```

Comment:
Idea is to normalize path after resolving it with respect to workspace folder.  Normalizing before would result in a in correct path for relative copybook path